src/codecatalyst/model.ts

```diff
@@ -287,13 +287,13 @@ export async function getDevfileLocation(client: DevEnvClient, root?: vscode.Uri
     async function checkDefaultLocations(rootDirectory: vscode.Uri): Promise<vscode.Uri> {
         // Check the projects root location
         const devfileRoot = vscode.Uri.joinPath(vscode.Uri.parse('/projects'), 'devfile.yaml')
-        if (await fileExists(devfileRoot.fsPath)) {
+        if (await fileOrFolderExists(devfileRoot.fsPath)) {
```

Comment:
is the rename needed? "file" is understood to mean any filesystem path, I think. That's presumably why the old name was `fileExists`.

"directory" is used for cases where non-directory files are expected.